src/platform/silabs/BlePlatformConfig.h

```diff
@@ -25,8 +25,6 @@
 
 #pragma once
 
-#include "sl_bt_api.h"
```

Comment:
Isn't this needed somewhere else then for the EFR32 build?

Reply:
EFR32 build seems to be building fine and the same include file can be used with the SiWx917